tutorials/lxd/use-cuda-within-lxd/use-cuda-within-lxd.md

```diff
@@ -0,0 +1,303 @@
+| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+|===============================+======================+======================|
+|   0  GeForce GTX 1080    Off  | 0000:01:00.0      On |                  N/A |
+| 43%   52C    P8    18W / 240W |    981MiB /  8110MiB |      1%      Default |
++-------------------------------+----------------------+----------------------+
+```
+
+## Install the CUDA toolkit
+Duration: 8:00
+
+With the drivers installed, the next step is to grab CUDA itself. While there are Ubuntu packages available, we've found the most reliable option is to download the latest version of CUDA directly from Nvidia's developer website:
+
+[https://developer.nvidia.com/cuda-downloads](https://developer.nvidia.com/cuda-downloads)
+
+To get to the download, select 'Linux', select your architecture, choose 'Ubuntu', '16.04' and finally the 'runfile (local)' installer type. You will then be able to download the base installer. In our example, this file is called `cuda_8.0.61_375.26_linux.run`.
+
+This file will need to be made executable and run with administrator privileges:
+
+```bash
+chmod +x cuda_8.0.61_375.26_linux.run
+sudo ./cuda_8.0.61_375.26_linux.run
+```
+
+The installer will ask a few questions.
+
+Accept the licence but answer `n` when asked whether you want to install the Nvidia accelerated driver. This is because the installer typically bundles an older version of the driver, such as 375.26 with our example, and you don't want this installed on-top of the driver we already have. All further questions can be answered `y` to accept their default values. 
+
+To add CUDA to your path, add the following to your bash configuration file, eg. `~/.bashrc`, and either re-source the file or log out and back in again:
+
+```bash
+export PATH=/usr/local/cuda-8.0/bin:$PATH
+export LD_LIBRARY_PATH=/usr/local/cuda-8.0/lib64:$LD_LIBRARY_PATH
+```
+
+Alternatively, add `/usr/local/cuda-8.0/lib64` to `/etc/ld.so.conf` and run 'ldconfig' as root.
+
+You can check CUDA is installed correctly by running `nvcc -V', which produces output similar to the following:
+
+```no-highlight
+nvcc: NVIDIA (R) Cuda compiler driver
+Copyright (c) 2005-2016 NVIDIA Corporation
+Built on Tue_Jan_10_13:22:03_CST_2017
+Cuda compilation tools, release 8.0, V8.0.61
+```
+
+Being correctly installed doesn't necessarily mean CUDA is correctly linked to the Nvidia driver. To make sure this is working, use the `bandwidthTest` utility from CUDA's demo_suite folder, usually found in '/usr/local/cuda-8.0/extras/demo_suite/'.
+
+If everything is working correctly, you should see `Result = PASS` at the end of the output. 
+
+## Launch LXD
+Duration: 5:00
+
+With the host now correctly configured and ready to go, it's time to launch LXD.
+
+If this is the first time you've used LXD, you will need to have a network and a storage pool defined. The `lxd init` command will step you through the process if you've not done this already.
```

Comment:
I think it would be okay to link to the LXD tutorial, or mention it and link to it at the end